app/android/AndroidManifest.xml

```diff
@@ -110,6 +110,9 @@
     <!-- Camera permissions -->
     <uses-permission android:name="android.permission.CAMERA"/>
 
+    <!-- We are reading sensors that from API 31 needs additional permission -->
+    <uses-permission android:name="android.permission.HIGH_SAMPLING_RATE_SENSORS" />
```

Comment:
https://developer.android.com/reference/android/Manifest.permission#HIGH_SAMPLING_RATE_SENSORS

how this connects to camera at all?

Reply:
We are reading OrientationSensor before taking a picture (for EXIF), here: https://github.com/lutraconsulting/input/blob/master/app/android/src/uk/co/lutraconsulting/CameraActivity.java#L49